### PR TITLE
add todo_list

### PR DESCRIPTION
**Pull request by DidenkoDenis306**
[DEMO LINK](https://DidenkoDenis306.github.io/react_static-list-of-todos/)

### CODE REVIEW

**Review by ruslanpashkov — APPROVED** [review]

File: src/components/TodoInfo/TodoInfo.tsx [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+  return (
+    <article className={`TodoInfo ${completed && 'TodoInfo--completed'}`}>
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      {user
+        && (
+          <UserInfo user={user} />
+        )}
```
Comment (33Orange) [inline src/components/TodoInfo/TodoInfo.tsx:22]:
```suggestion
      {user && (
          <UserInfo user={user} />
        )}
```

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { completed, title, user } = todo;
+
+  return (
+    <article className={`TodoInfo ${completed && 'TodoInfo--completed'}`}>
```
Comment (33Orange) [inline src/components/TodoInfo/TodoInfo.tsx:14]: `completed && 'TodoInfo--completed'` this condition will be add 'false' as className if `completed === false
`

Try something else. 

image

**Review by 33Orange — CHANGES_REQUESTED** [review]
Good job. Almost done. 

A few moments to fix

**Review by 33Orange — APPROVED** [review]
Well done !

**Pull request closed without merging**